sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/_input_output_helpers.py

```diff
@@ -332,12 +332,16 @@ def to_rest_data_outputs(outputs: Optional[Dict]) -> Dict[str, RestJobOutput]:
 
                 output_value_type = output_value.type if output_value.type else AssetTypes.URI_FOLDER
                 if output_value_type in target_cls_dict:
+                    if getattr(output_value, "pathOnCompute", None) is not None:
+                        sourcePathOnCompute = output_value.pathOnCompute
+                    else:
+                        sourcePathOnCompute = None
                     output = target_cls_dict[output_value_type](
                         asset_name=output_value.name,
                         asset_version=output_value.version,
                         uri=output_value.path,
                         mode=OUTPUT_MOUNT_MAPPING_TO_REST[output_value.mode.lower()] if output_value.mode else None,
-                        pathOnCompute=output_value.path_on_compute if output_value.path_on_compute else None,
+                        pathOnCompute=sourcePathOnCompute,
```

Comment:
This can be simplified as 

pathOnCompute=getattr(output_value, "pathOnCompute", None)

Reply:
thanks I have followed you suggestions